content/sensu-go/5.0/guides/install-check-executables-with-assets.md

```diff
@@ -11,153 +11,117 @@ menu:
---command "sensu-influxdb-handler -d sensu" \
---env-vars "INFLUXDB_ADDR=http://influxdb.default.svc.cluster.local:8086, INFLUXDB_USER=sensu, INFLUXDB_PASSWORD=password"
+sensuctl asset list
 {{< /highlight >}}
 
-### Validating the handler asset
-
-Once the handler is setup, it must be attached to a check to handle the event output (ex. `collect-metrics`).
+### 3. Create a workflow
 
-{{< highlight shell >}}
-$ sensuctl check set-handlers collect-metrics influx-db
-{{< /highlight >}}
+Now we can use assets in a monitoring workflow.
+Depending on the asset, you may want to create Sensu checks, filters, mutators, and handlers.
+The asset details in Bonsai are the best resource for information about asset capabilities and configuration.
 
-You can verify that the handler has fetched/installed the asset, and executed the dependency by checking the
-backend logs.
+For example, to use the [Sensu PagerDuty handler asset][19], create a `pagerduty` handler that includes your PagerDuty service API key in place of `SECRET` and `sensu-pagerduty-handler` as a runtime asset.
 
 {{< highlight json >}}
-{"assets":["sensu-influxdb-handler"],"component":"pipelined","namespace":"default","handler":"influx-db","level":"debug","msg":"fetching assets for handler","time":"2018-10-16T13:17:33-07:00"}
+{
+    "api_version": "core/v2",
+    "type": "Handler",
+    "metadata": {
+        "namespace": "default",
+        "name": "pagerduty"
+    },
+    "spec": {
+        "type": "pipe",
+        "command": "sensu-pagerduty-handler",
+        "env_vars": [
+          "PAGERDUTY_TOKEN=SECRET"
+        ],
+        "runtime_assets": ["sensu-pagerduty-handler"],
+        "timeout": 10,
+        "filters": [
+            "is_incident"
+        ]
+    }
+}
 {{< /highlight >}}
-{{< highlight json >}}
-{"component":"pipelined","namespace":"default","handler":"influx-db","level":"info","msg":"event pipe handler executed","output":"metric sent to influx-db","status":0,"time":"2018-10-16T13:17:33-07:00"}
-{{< /highlight >}}
-
-## How to create a mutator that depends on an asset 
-
-### Adding an asset to a mutator on creation
 
-[Create an asset][2] (ex. `transformer`), and create a mutator (ex. `transform-metrics`).
+Save the definition to a file, and add to Sensu using sensuctl.
 
 {{< highlight shell >}}
-sensuctl mutator create transform-metrics \
---command "transform --type metrics" \
---runtime-assets transformer
+sensuctl create --file filename.json
 {{< /highlight >}}
 
-### Validating the mutator asset
+Now that Sensu can create incidents in PagerDuty, we can automate this workflow by adding the `pagerduty` handler to our Sensu service checks.
+To get started with checks, see the [guide to monitoring server resources](../monitor-server-resources).
 
-Once the mutator is setup, it must be attached to a handler to mutate the event output (ex. `influx-db`).
+You can use sensuctl to see available checks, handlers, and other Sensu resources.
```

Comment:
This feels somewhat disconnected from the rest of what's described here about assets.

Perhaps we move the "to get started with checks" line down to Next steps and get rid of the content about using sensuctl to list available resources?